packages/medusa/src/models/order.js

```diff
@@ -13,8 +10,12 @@ class OrderModel extends BaseModel {
   static modelName = "Order"
 
   static schema = {
```

Comment:
We probably need some tracking numbers as well - unless we can somehow put that in the shipping methods. Let me know what you think

Reply:
I think tracking numbers is a good idea. Maybe we should add an array of tracking numbers, in which we include shipping provider name, id and tracking number for each of them.